analytics/management/commands/update_analytics_counts.py

```diff
@@ -36,6 +40,19 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         # type: (*Any, **Any) -> None
+        LOCK_DIR = "/tmp/update_analytics_lock"
+        try:
+            os.mkdir(LOCK_DIR)
+        except OSError:
+            print(WARNING + "cronjob in progress; waiting for lock... " + ENDC)
```

Comment:
The py3k static checker error is that you need to add `from __future__ import print_function` at the top of this file.